docs/articles/new-expensify/getting-started/Security.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Security
+
```

Comment:
You'll need to add a `description: ` here as well, with a short phrase about the page

Reply:
@dangrous @rushatgabhane I updated the issue and included the description. Can you please confirm if what I did is correct? Is there anything else I need to do before this article is published? Thanks!